pom.xml

```diff
@@ -818,6 +818,16 @@
             <groupId>com.digi.xbee</groupId>
             <artifactId>xbee-java-library</artifactId>
             <version>1.2.0</version>
+            <exclusions>
+               <exclusion> 
+                  <groupId>org.slf4j</groupId>
+                  <artifactId>slf4j-log4j12</artifactId>
```

Comment:
You probably want to block the slf4j-jdk14 dependency instead of slf4j-log4j (we *want* slf4j-log4j since that's what we use internally). I don't think that blocking log4j itself is needed either.